assets/dns/daemonset.yaml

```diff
@@ -150,6 +150,10 @@ spec:
       dnsPolicy: Default
       nodeSelector:
         kubernetes.io/os: linux      
+      updateStrategy:
+        type: RollingUpdate
+        rollingUpdate:
+          maxUnavailable: 10%
```

Comment:
TIL that `maxUnavailable` values specified by percentage are rounded _up_ [for Daemonsets](https://github.com/kubernetes/api/blob/master/apps/v1/types.go#L488-L491) and _down_ [for Deployments](https://github.com/kubernetes/api/blob/master/apps/v1/types.go#L347-L349). In other words, setting `maxUnavailable: 10%` here is effectively setting `maxUnavailable: 1` for any cluster with <= 10 nodes (such as our CI clusters) as opposed to `maxUnavailable: 0`.

Would it be worth adding in a comment here to clarify that the calculated percentage is rounded up?

Reply:
Comment added.